.github/actions/releases/setup-android/action.yml

```
@@ -32,6 +32,13 @@ runs:
 
     - name: Setup Android SDK
       uses: android-actions/setup-android@v3
+      
+    # "Fixes firebase build error when no valid google-services.json is present"
+    - name: Comment out Google Play Services import
+      shell: bash
+      run: |
+        sed -i 's/id .com\.google\.gms\.google-services./\/\/ id .com\.google\.gms\.google-services./' android/app/build.gradle
+        echo "Google Play Services import commented out"
```

Release builds should fail if Firebase is not configured correctly. I will fix the root cause shortly. This also happens with macOS or iOS (can't recall which it was). 

Please also create tasks for (normal priority):
1. Ensure this behaviour is consistent across platforms.
2. Amend our Firebase local setup documentation to include instructions and/or a script in our CLI package script to "de-Google" the builds.